flows/java/src/main/java/io/rapidpro/flows/definition/actions/contact/SaveToContactAction.java

```diff
@@ -51,6 +56,21 @@ else if (m_field.equals("first_name")) {
                 label = "First Name";
                 run.getContact().setFirstName(value);
             }
+            else if (m_field.equals("tel_e164")) {
+                value = StringUtils.substring(value, 0, 128);
+                label = "Phone Number";
+
+                List<ContactUrn.Scheme> schemes = new ArrayList<>();
+                schemes.add(ContactUrn.Scheme.TEL);
+                ContactUrn urn = run.getContact().getUrn(schemes);
```

Comment:
Only Java designers would prefer N factory methods to just adding @#% list literals already